docs/reference/configuration-parameters.rst

```diff
@@ -4,6 +4,24 @@ Configuration Parameters
 
 This section contains a list of properties that can be configured in ``scylla.yaml`` - the main configuration file for ScyllaDB.
 In addition, properties that support live updates (liveness) can be updated via the ``system.config`` virtual table or the REST API.
+So configuration settings are located in multiple places. Certain locations take precedence over others. ScyllaDB's configuration
+settings take precedence in the following order:
+
+============== =======================
+precedence     location of parameters
+============== =======================
+1              runtime update
```

Comment:
Not clear what this `runtime update` is.

Reply:
they are SIGHUP and API. will elaborate it a little bit.